<launch>
    <include file="$(find reelrbtx_demos)/launch/simulated_robot_simulated_reel.launch">
        <arg name="rqt_plot" value="false"/>
    </include>
    
    <node pkg="radial_motion_example" type="radial_motion_example_node" name="RadialMotionSample" launch-prefix="xterm -hold -e "/>
    
    <!-- smacc action servers -->
    <node pkg="non_rt_helper" type="non_rt_helper_node" name="non_rt_helper" launch-prefix="xterm -hold -e "/>
    <node pkg="smacc_tool_plugin_template" type="tool_action_server_node" name="tool_action_server_node" launch-prefix="xterm -hold -e "/>
</launch>